### DISCUSSION TITLE
Error on Port 1337 when running cs50 style .

### DISCUSSION THREAD

**Discussion by AlfredoZimmermann** [community/community#64252] [Codespaces]
### Select Topic Area

Bug

### Body

When running style it's given me failed to connect extention on port 1337 . I had to make another codespace  to do it, but now I can't create another codespace and I noticed when I managed to run it it was showing 7 ports, now it's showing just 5 without the 1337.

**Comment by MasovichG** [comment]
Try full rebuild 
After installing “GitHub Codespaces” extension, press “Command + Shift + P” (if on macOS) or “Ctrl + Shift + P” (if on Windows), and search for “full rebuild”, select “Full Rebuild Container” to perform a codespace rebuild. Your codespace will be launched once the rebuild process completes.

**Reply by khalil-ur** [reply]
Command 'Codespaces: Rebuild Container' resulted in an error
command 'github.codespaces.rebuildEnvironment' not found

**Reply by Ghanshyam-Goley** [reply]
@MasovichG Man you are truly a lifesaver. Thank you from heart

**Comment by Alexayechanzay** [comment]
That's work !

**Comment by Mehdi-Rj** [comment]
worked for me too!

**Comment by Xelliese** [comment]
Thanks brother, worked here too.
